### DISCUSSION TITLE
Will not run nvim only offers VIM

### DISCUSSION THREAD

**Discussion by thorrrr** [LazyVim/LazyVim#458] [Q&A] [answered]
HI Guys
On Fedora 377 KDE tried you easy install instructions but it fails 

![image](https://user-images.githubusercontent.com/8170673/226123152-e26829ee-135c-4249-a912-f72a601eed69.png)

**Comment by loiccoyle** [comment, marked as answer]
`LazyVim` is a `neovim` config, so you need to install `neovim`, see https://neovim.io/.
